ui/components/multichain/account-list-item/account-list-item.js

```diff
@@ -163,12 +169,15 @@ const AccountListItem = ({
     formattedTokensWithBalancesPerChain,
   );
   // cross chain agg balance
+  // Yeah this is hacky... do we really need balances here just to grab icons?
+  const chainIconsStr = JSON.stringify(
+    accountTotalFiatBalances.orderedTokenList
+      .map((i) => i.iconUrl)
+      .filter(Boolean),
+  );
   const mappedOrderedTokenList = useMemo(
-    () =>
-      accountTotalFiatBalances.orderedTokenList.map((item) => ({
-        avatarValue: item.iconUrl,
-      })),
-    [accountTotalFiatBalances.orderedTokenList],
+    () => JSON.parse(chainIconsStr).map((url) => ({ avatarValue: url })),
+    [chainIconsStr],
```

Comment:
It's a little nuts that the total fiat balance impacts the tokens urls we show.
This is a little hack to prevent this token list from continuously changing. We can remove.

Reply:
I think that we actually have `useTokenBalances` hook that can compute balances for us, rather than relying on this hook.

This hook was one of Brian's final contributions on the way out. He also identified that we shouldn't need to compute balances here to get icons, and that they should be handled as two separate operations.

We never fully integrated `useTokenBalances` everywhere though

Reply:
Ooooh awesome!! Will make a note of this!